.travis.yml

```diff
@@ -43,6 +43,5 @@ script:
         source ./.travis_scripts/create_feedstocks;
       else
         echo "Building all recipes.";
-        source ./.travis_scripts/build_all;
+        travis_wait 60 source ./.travis_scripts/build_all;
```

Comment:
This needs to be removed before merging. On the upstream feedstock, we will have to add this fix back I think. Or maybe switch to a circleci osx build and let the build be more noisy.